HelpAssistHer/help-assist-her
# Add PrimaryContactPerson, closedAllDay, bug fixes in front-end/back-end testing, new data for testing

## Description
* changes `PrimaryContact` to `PrimaryContactPerson` in all cases
* adds in `closedAllDay` as part of `hours` (`open-now` endpoint will need to change in the future to accommodate this)
* updates the data import process to match new schemas (`yarn data`)
* rewrites `database-helpers.js` to fix issues that were preventing the front-end from saving `primaryContactPerson` and `verifiedData` correctly, with added bonus of better readability
* adds a new command for front-end/back-end testing: `yarn test-data`. `yarn test-data` drops everything from the database and imports one pregnancy center that has nearly all the possible data filled in with fake data, so that it's easy to see what isn't displaying on the front-end, and then also, what isn't saving on the backend. 

## Test Plan
`yarn test`

ALSO

1. `yarn test-data`
2. Go to `http://localhost:4000/verification`
3. Login
4. Get resource to verify
5. Check that all fields are filled in
6. Change fields
7. Save
8. Reload `http://localhost:4000/verification`
9. Get resource again
10. Check that you see the changes you made
